### ISSUE TITLE
Crash when no titles in the input DAT match your preferences

### ISSUE THREAD

**Issue by AlvaroBurnett** [unexpectedpanda/retool#300]
When running the command line retool, having filled user-config.yaml in a way that the resulting dat will turn empty (for example, asking for spanish titles in a WonderSwan dat), the program works but in the end it crashes with the following message:

```
Traceback (most recent call last):
  File "S:\ssd\host\programs\offline\retool\program-cli\retool.py", line 309, in <module>
    main()
  File "S:\ssd\host\programs\offline\retool\program-cli\retool.py", line 284, in main
    f'{str("{:,}".format(stats_final_count))} '
                         ^^^^^^^^^^^^^^^^^
UnboundLocalError: cannot access local variable 'stats_final_count' where it is not associated with a value
```

**Comment by unexpectedpanda** [comment]
It should never access the stats count if there's no titles found that match your preferences, so that's a bit odd.

I can't replicate this at this stage. Can you provide the following details for me to test?

- Your `config/user-config.yaml` file.
- The DAT file that causes the problem.
- The command line arguments you're using.

**Comment by unexpectedpanda** [comment]
Accidental close, sorry.

**Comment by unexpectedpanda** [comment]
Fixed in v2.02.1.
